app/src/features/mocks/screens/mocksList/components/MocksList/components/MocksTable/utils/index.ts

```diff
@@ -1,11 +1,11 @@
 import { MockRecordType, RQMockMetadataSchema } from "components/features/mocksV2/types";
 import { MockTableRecord } from "../types";
 
-export const isRecordMock = (record: RQMockMetadataSchema) => {
+export const isMock = (record: RQMockMetadataSchema) => {
   return !record.recordType || record.recordType === MockRecordType.MOCK;
 };
 
-export const isRecordMockCollection = (record: RQMockMetadataSchema) => {
+export const isCollection = (record: RQMockMetadataSchema) => {
   return record.recordType === MockRecordType.COLLECTION;
 };
```

Comment:
[this comment](https://github.com/requestly/requestly/pull/2123#discussion_r1742758683) seems to have been removed

Even though you have given an explanation of the cause [here](https://github.com/requestly/requestly/pull/2123#discussion_r1743271889), it does not justify NOT FIXING this right now